articles/tech-lead-needed.md

```diff
@@ -0,0 +1,98 @@
+---
+authorName: Daniel Mölle
+authorGithubUsername: dmlfra
+title: Your team needs a Tech Lead, not a Lead Techie
+---
+# {{page.title}}
+
+In what follows, let us just assume that a tech lead is an experienced software engineer who is simultaneously (a) supposed to lead the development team and (b) responsible for the entire technical solution.
+
+I know that this is not the only valid interpretation. In the same way that different organizations have different habits with respect to setting up project teams, there is no universal definition for project roles such as *tech leads* or *software architects*. Moreover, I am well aware of the fact that many agile methodologies even discourage us from assigning such singular roles to individual team members (even though they often exist implicitly in that case). However, I can only write about my personal experience -- more than ten years in device development projects at Zühlke Frankfurt, all of which had a tech lead in the very sense outlined above.
```

Comment:
I'd skip this complete paragraph. It does not add additional information but looks like you're preparing against some attacks.

Or at least shorten it to something like: "Although there are a lot of other valid interpretations available, this is the one applied in most of the projects I worked in."